feature/storage/src/main/java/com/mashup/dorabangs/feature/storage/storagedetail/StorageDetailViewModel.kt

```diff
@@ -219,28 +222,24 @@ class StorageDetailViewModel @Inject constructor(
     }
 
     /**
-     * 즐겨찾기 추가
-     * 낙관적 Update추가, 스크롤 위치가 왜 변할까?
+     * 즐겨찾기 추가 - 낙관적 업데이트 적용
      */
     fun addFavoriteItem(postId: String, isFavorite: Boolean) = viewModelScope.doraLaunch {
-        intent {
-            val postInfo = PostInfo(isFavorite = !isFavorite)
-            patchPostInfoUseCase(
-                postId = postId,
-                postInfo = postInfo,
-            )
-            // intent { postSideEffect(StorageDetailSideEffect.RefreshPagingList) }
-            reduce {
-                val updateList = state.pagingList.map { pagingData ->
-                    pagingData.map { item ->
-                        if (item.postId == postId) {
-                            item.copy(isFavorite = !isFavorite)
-                        } else {
-                            item
-                        }
-                    }
+        _feedListState.value = feedListState.value.map { item ->
+            if (item.postId == postId) {
+                item.copy(isFavorite = !isFavorite)
+            } else {
+                item
+            }
+        }
+        val isSuccessFavorite = patchPostInfoUseCase(postId = postId, postInfo = PostInfo(isFavorite = !isFavorite)).isSuccess
+        if (!isSuccessFavorite) {
```

Comment:
이거 .not()쓰면 . 더명확해보여요 
```suggestion
        if (isSuccessFavorite.not()) {
```

Reply:
변경!